### PR TITLE
stolostron/backlog#20964 cim alt flow

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/create_cluster_on_prem.adoc [hunk 1]
```diff
@@ -26,7 +26,14 @@ To create clusters from the {product-title-short} console, complete the followin
 
 . From the navigation menu, navigate to *Infrastructure* > *Clusters*.
 . On the _Clusters_ page, click *Create cluster*.
-. Select *On-premises* for the infrastructure provider. The remaining steps are based on your selection.
+. Select one of the following options for your assisted installation: 
++
+* *Use existing discovered hosts*: Select your hosts from a list of hosts that are in an existing infrastructure environment.
+* *Discover new hosts*: Retrieve a list of available hosts that might or might not be in an existing infrastructure environment.
```
Comment (jelkosz) [inline clusters/create_cluster_on_prem.adoc]: nope, the hosts dont exist at this point yet. Id write something like: 
Dont use the hosts which are already present in an infrastructure environment, discover your own hosts instead.

**Comment by jelkosz** [comment]
just one comment, otherwise looks good!

**Comment by jelkosz** [comment]
just one comment, otherwise looks good!

**Comment by cadawson** [comment]
Thanks @jelkosz . I don't see any new comments from you in the issue or the PR. Did you merge it into the PR already? I just want to make sure I don't miss it.

**Review by jelkosz — CHANGES_REQUESTED** [review]

**Comment by jelkosz** [comment]
sorry @cadawson , forgot to submit them :(

**Comment by jelkosz** [comment]
lgtm

**Pull request merged**